### ISSUE TITLE
PDF Bookmark structure incorrect when there is only 1 part and it is empty

### ISSUE THREAD

**Issue by josieg** [pressbooks/pressbooks#2300]
### Description

When a book has only one part and that part has no content, the bookmark structure in the Digital and Print PDFs is incorrect, with the chapters being grouped under the last front matter rather than being on the same level. 
Here is when there is text in the "Main body" part:
![When Main Body part is not empty](https://user-images.githubusercontent.com/23706762/128388439-dd4df828-9ead-443a-a9b0-d818718ea707.jpg)
Here is where there is no text in the "Main body" part:
![When Main Body part is empty](https://user-images.githubusercontent.com/23706762/128388454-f6693827-20c7-4e7d-9829-3989966cb6e3.jpg)

### Steps to Reproduce

1. Create a new book. It should have only one part and there should be no content in the part.
2. Export the book as a PDF (Digital or print, doesn't matter).
3. Open the PDF file and view the bookmarks.

**Expected behavior:** 
I would expect that when a book does not use any parts (i.e., parts do not show up in the table of contents), that the chapters should be the same level as the front and back matter in the PDF bookmarks.

**Actual behavior:** 
The bookmarks make it look like the book's chapters are subsections of the last front matter.

Same issue as https://github.com/pressbooks/pressbooks-book/issues/598

**Comment by kqzheng** [comment]
When a Part has H1, H2 or H3, it affects the exported PDF structure. When an author was selected in the the metadata of a book, the author's name would display in the exported PDF structure at the same level with the chapter title. 
![image](https://user-images.githubusercontent.com/88904178/129406317-212968ed-ef77-4308-959d-a54caa5e11ce.png)


Test book: https://kpu.pressbooks.pub/testkz/ 

**Steps to reproduce:**

- Create a part
- Edit the Part, and add H1, H2 or H3 in the Part content
- Create a chapter
- In the metadata, add a Chapter Author from the list of contributor
- Save and export PDF

**Expected behaviour:**

- Chapters in a part collapse under the "Part"
- Headings in a Part do **not** display in the PDF structure
- Author's name do **not** display in the PDF structure

**Actual behaviour:**

- In the PDF bookmark structure, H1 and H2 in a Part display at the same level as the Part, H3 in a part displays at the same level as the Chapter in the Part.
- In the PDF bookmark structure ,Author's name of the chapter displays at the same level as the chapter.
